### ISSUE TITLE
Recommended grammar/spelling change (English)

### ISSUE THREAD

**Issue by jfburke3** [osmandapp/Osmand#4902]
Recent night builds appear to have introduced a helpful note on POI detail screens about when they will open, or be open until.  However, the chosen spelling sounds awkward.

In this example, the note reads "Will be opened at 16:30":

![will be opened](https://user-images.githubusercontent.com/11140831/34644614-d8efe0ae-f307-11e7-9a99-4e21de8f3434.jpg)

The use of passive voice ("will be"), and past tense ("opened"), just doesn't sound right.  A better way to word it would be "Will open at...."

Similarly, places that are currently open show "Opened till...." where a better choice would be "Open till" or "Open until" (the latter is more proper, but the former isn't uncommon).

![opened till](https://user-images.githubusercontent.com/11140831/34644648-40f5be6c-f308-11e7-84da-4e326fc151f3.jpg)

**Comment by jfburke3** [comment]
Thanks guys, that looks better.

I want to make one additional observation, but I'm not going to open a new issue on it because it might simply be a US/UK difference.

The new text "Will open on 11:30 We" isn't US English.  We would say "Will open at 11:30 We" but I know that in the UK they sometimes use prepositions differently than we do, and I'm not sure if this wouldn't be correct for them.

**Comment by sonora** [comment]
Yes, precisely. I have some years ago opened a UK resource file for any deviations like that, while in our main strings files we should strictly implement the US versions. 

I have fixed this particular string already. Please keep an eye on new strings, many of our developers are not native speakers, and I am not always the quickest to review all the new strings.

Regarding "on 11:30 Wed": I believe that to be wrong in any case, everywhere  :)

Regarding our general inflation of new strings: I think we are exaggerating here, often introducing new strings where existing ones should suffice. But will have to address that elsewhere. @vshcherb : Maybe we can advise our developers to better which new strings are really needed vs. what can be reused, it could both produce better app quality and less inflated apk's?

**Comment by sonora** [comment]
Let me re-open this as a reminder, "Will open on 11:30 Wed"seems still not fixed, issue likely in the code anf not in the strings.

**Comment by sonora** [comment]
I have looked at the code and it seems there are no cases where you would just have a day, there is always a time preceding. Hence I changed the corresponding string `will_open_on` to contain "Will open at", which is correct English. "on" would only be used  in the absence of any clock time specification.

But having a separate string for this time+day case may be needed in other languages, hence we should leave it in place.
